recipe_sha: be414f7cf31d27150bc38d37d07ac334f2f45051e05b9d90f09e93127238913e  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Attempting to finalize metadata for ucsc-parahub
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['ucsc-parahub-469-hd4000eb_1.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/ucsc-parahub_1733853560038/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_


  The following NEW packages will be INSTALLED:

      _openmp_mutex:     4.5-2_gnu            conda-forge
      ca-certificates:   2024.8.30-hcefe29a_0 conda-forge
      libgcc:            14.2.0-he277a41_1    conda-forge
      libgcc-ng:         14.2.0-he9431aa_1    conda-forge
      libgomp:           14.2.0-he277a41_1    conda-forge
      libopenssl-static: 3.4.0-h86ecc28_0     conda-forge
      libpng:            1.6.44-hc4a20ef_0    conda-forge
      libstdcxx:         14.2.0-h3f4de04_1    conda-forge
      libstdcxx-ng:      14.2.0-hf1166c9_1    conda-forge
      libuuid:           2.38.1-hb4cce97_0    conda-forge
      libzlib:           1.3.1-h86ecc28_2     conda-forge
      mysql-connector-c: 6.1.11-h8945da3_1008 conda-forge
      openssl:           3.4.0-h86ecc28_0     conda-forge
      zlib:              1.3.1-h86ecc28_2     conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_build_env): ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_test_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/ucsc-parahub_1733853560038/_build_env


  The following NEW packages will be INSTALLED:

      _openmp_mutex:                4.5-2_gnu           conda-forge
      binutils_impl_linux-aarch64:  2.43-h4c662bb_2     conda-forge
      binutils_linux-aarch64:       2.43-hf1166c9_2     conda-forge
      gcc_impl_linux-aarch64:       13.3.0-hcdea9b6_1   conda-forge
      gcc_linux-aarch64:            13.3.0-h1cd514b_7   conda-forge
      kernel-headers_linux-aarch64: 4.18.0-h05a177a_18  conda-forge
      ld_impl_linux-aarch64:        2.43-h80caac9_2     conda-forge
      libgcc:                       14.2.0-he277a41_1   conda-forge
      libgcc-devel_linux-aarch64:   13.3.0-h0c07274_101 conda-forge
      libgomp:                      14.2.0-he277a41_1   conda-forge
      libsanitizer:                 13.3.0-ha58e236_1   conda-forge
      libstdcxx:                    14.2.0-h3f4de04_1   conda-forge
      make:                         4.4.1-h2a6d0cb_2    conda-forge
      sysroot_linux-aarch64:        2.17-h5b4a56d_18    conda-forge
      tzdata:                       2024b-hc8b5060_0    conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Source cache directory is: /opt/conda/conda-bld/src_cache
  Downloading source to cache: userApps.v469.src_3f95b26ccc.tgz
  Downloading http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v469.src.tgz
  WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0xee34113c3df0>, 'Connection to hgdownload.cse.ucsc.edu timed out. (connect timeout=9.15)')': /admin/exe/userApps.archive/userApps.v469.src.tgz
  WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0xee34113c2860>, 'Connection to hgdownload.cse.ucsc.edu timed out. (connect timeout=9.15)')': /admin/exe/userApps.archive/userApps.v469.src.tgz
  WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0xee34113c3910>, 'Connection to hgdownload.cse.ucsc.edu timed out. (connect timeout=9.15)')': /admin/exe/userApps.archive/userApps.v469.src.tgz
  WARNING: Error: HTTP 000 CONNECTION FAILED for url <http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v469.src.tgz>
  Elapsed: -

  An HTTP error occurred when trying to retrieve this URL.
  HTTP errors are often intermittent, and a simple retry will get you on your way.
  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3712, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2449, in build
      try_download(m, no_download_source=False, raise_error=True)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/render.py", line 762, in try_download
      source.provide(metadata)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 1035, in provide
      unpack(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 169, in unpack
      src_path, unhashed_fn = download_to_cache(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 116, in download_to_cache
      raise RuntimeError(f"Could not download {url}")
  RuntimeError: Could not download http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v469.src.tgz
# Last 100 lines of the build log.
